### ISSUE TITLE
home-cctv

### ISSUE THREAD

**Issue by Mariusthvdb** [Templarian/MaterialDesign#6863]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

to replace the current icon cctv with one depicting a home, and be consistent in the frontend:

<img width="242" alt="Scherm­afbeelding 2022-11-30 om 16 29 25" src="https://user-images.githubusercontent.com/33354141/204840471-48d5f40d-a6c8-4922-a3e1-1f4022822c9b.png">

### Give us an example of what this icon might look like.

just like the other home- icons: a home and an overlay

**Comment by terreng** [comment]
![image](https://user-images.githubusercontent.com/11605395/204903375-53d05349-c332-40c7-857e-7ec3ad9810d4.png)
[home-video.svg](https://github.com/terreng/samples/blob/master/home-video.svg)

![image](https://user-images.githubusercontent.com/11605395/204904155-35b3a3c0-5846-41c4-8473-0a96c758643e.png)
[home-video-outline.svg](https://github.com/terreng/samples/blob/master/home-video-outline.svg)

**Comment by Mariusthvdb** [comment]
Cool! Maybe even better than the corner cutout ;-)

Did you try that too?

**Comment by terreng** [comment]
I found the cctv icon to be a bit too detailed to be shrunk down into a corner.

**Comment by Mariusthvdb** [comment]
its a bit difficult to spot in a darker theme in the left menu item:

<img width="342" alt="Scherm­afbeelding 2022-11-30 om 22 47 57" src="https://user-images.githubusercontent.com/33354141/204915391-8cb6e1e6-d55b-4d59-8137-fe0512683c41.png">

though accenting it in gold color does help a lot.

in a lighter theme:

<img width="244" alt="Scherm­afbeelding 2022-11-30 om 22 48 45" src="https://user-images.githubusercontent.com/33354141/204915584-35c8ff7a-04ce-4030-8029-bb959c14aca1.png">

its much better.

and still, the icon itself (and the others) seems significantly smaller than the home-assistant icon, which does not help.

**Comment by terreng** [comment]
Yeah, the `home-assistant` icon is also the only home icon with a chimney. That might be worth a separate issue.

**Comment by mririgoyen** [comment]
> Cool! Maybe even better than the corner cutout ;-)
> 
> Did you try that too?
> 
> I'll test these a bit later and report the dashboard view for your info

CCTV is not an icon modifier, so we wouldn't accept it used as such in the library.

**Comment by Mariusthvdb** [comment]
is there any there any other modifier you would except for this FR? 
Or are you only talking about the corner cut-out, and would the current version above be acceptable.

**Comment by Mariusthvdb** [comment]
since this is a bit in limbo still (and using the custom home-cctv with great pleasure ever since above) please let me ask if this will be included?

if so, I'd love to asl for addition of an -off and -settings version too. (just add the regular diagonal line and cogwheel respectively)

These would fit nicely and give us very useful feedback options in the Frontend of our system
thanks for considering
